src/main/java/service/OrderService.java

```diff
@@ -49,4 +48,16 @@ public Order findOrder(final String orderNumber) {
 				.findFirst()
 				.orElse(null);
 	}
+
+	public boolean changeOrderStatus(final String orderNumber, final OrderStatus newStatus) {
+		final Order order = findOrder(orderNumber);
+		if (order != null) {
+			order.setOrderStatus(newStatus);
+			System.out.println("Zmieniono status zamówienia o numerze " + orderNumber + " na: " + newStatus);
```

Comment:
Get rid of souts here. Service cannot communicate with users, it's the menu's responsibility.